### PR TITLE
Add the assets page

### PR DESCRIPTION
**Pull request by yongjiema**

### CODE REVIEW

File: apps/neoscan/lib/neoscan/counters/counters.ex [hunk 1]
```diff
@@ -4,11 +4,17 @@ defmodule Neoscan.Counters do
...
   alias Neoscan.Repo
   alias Neoscan.Counter
+  alias Neoscan.AddressBalance
+  alias Neoscan.AddressTransactionBalance
 
   def count_addresses do
     Repo.one(from(c in Counter, where: c.name == "addresses", select: c.value))
   end
 
+  def count_addresses(asset_hash) do
```
Comment (adrienmo) [inline apps/neoscan/lib/neoscan/counters/counters.ex]: I am afraid this part will not scale well with the production database, we should probably use a trigger to populate a counter.

File: apps/neoscan/lib/neoscan/counters/counters.ex [hunk 2]
```diff
@@ -34,4 +40,18 @@ defmodule Neoscan.Counters do
       0
     ]
   end
+
+  def count_transactions(asset_hash) do
+    Repo.one(
```
Comment (adrienmo) [inline apps/neoscan/lib/neoscan/counters/counters.ex]: Same comment than for the count_addresses

File: apps/neoscan/lib/neoscan/counters/counters.ex (hunk 1 above)
Comment (yongjiema) [reply apps/neoscan/lib/neoscan/counters/counters.ex]: Thanks for your feedback, I will update it.

**Comment by yongjiema** [comment]
Updated, please review again, thanks!

**Comment by adrienmo** [comment]
@yongjiema looks good, I am going to merge it in a branch, and I will set it aside until I have the opportunity to resync neoscan to integrate this change :)

**Pull request merged**